README.md

```diff
@@ -2,5 +2,62 @@
 
 Terraform module that will create an S3 bucket in AWS with relevant user account that will have access to bucket.
 
+The bucket created will use $team_name-$bucket_identifier as the bucket name to ensure the bucket created is globally unique and avoid name clashes with buckets.
```

Comment:
Reading this made me realise that given the global S3 bucket namespace, our naming convention is probably *not* good enough to avoid clashes. We might want to prefix with yet another identifier (eg. `moj-cp-`)

@kalbir @kerin thoughts?

Reply:
maybe i can prefix the business unit from the tag with team name and bucket identifier.

I.E mojdigital-${team_name}-${bucket_identifier}
     hmcts-${team_name}-${bucket_identifier}

Reply:
discussed witj Kalbir, using business-unit as a prefix.

Such as ${business-unit}-${team_name}-${bucket_identififer}